charts/renku/requirements.yaml

```diff
@@ -29,6 +29,6 @@ dependencies:
   repository: "https://charts.bitnami.com/bitnami"
   condition: postgresql.enabled
 - name: keycloak
-  version: 4.10.1
-  repository: "@stable"
+  version: 4.10.2
```

Comment:
why the change in this PR?

Reply:
> why the change in this PR?

Because in the new codecentric repo that's the oldest chart available. The deployment does not work otherwise, so it is indeed a small inevitable update. Luckily, this is just a bugfix change in versions so it should not break things. Authorization seem to work just fine in that deployment.
It seems that we might want to work on #1003 sooner rather than later.

Reply:
I was going to push the very same changes cause I had the issue with `keycloack` :smile: 

Some extra references for the reviewers:
* Deprecated: https://github.com/helm/charts
* New source: https://github.com/codecentric/helm-charts
* Problem with the version:
```
helm search repo keycloak --versions
NAME            CHART VERSION   APP VERSION     DESCRIPTION                                       
stable/keycloak 4.10.1          5.0.0           DEPRECATED - Open Source Identity and Access Ma...
stable/keycloak 4.10.0          5.0.0           Open Source Identity and Access Management For ...
```